cpg-language-typescript/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/typescript/TypeScriptLanguageFrontend.kt

```diff
@@ -71,20 +71,30 @@ class TypeScriptLanguageFrontend(
     private val mapper = jacksonObjectMapper()
 
     companion object {
-        private val parserFile: File = createTempFile("parser", ".js")
+        private val parserFile: File = createTempFile("parser", "")
 
         init {
-            val link = this::class.java.getResourceAsStream("/nodejs/parser.js")
+            val arch =
+                System.getProperty("os.arch").replace("aarch64", "arm64").replace("x86_64", "amd64")
```

Comment:
Why do we make these replacements instead of simply naming the binaries correctly in the `build.gradle.kts`?

Reply:
Good point, I can rename the binary